Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE_Airburst.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Verse;
+using RimWorld;
+using CombatExtended.Utilities;
+
+namespace CombatExtended
+{
+    public class ProjectileCE_Airburst : ProjectileCE
+    {
+        public int updateRate => props.updateRate;
+
+        public int timer = -10;
+        
+        public bool detonated;
+
+        public ModExtension_Airburst props => this.def.GetModExtension<ModExtension_Airburst>();
+        public override void Tick()
+        {
+            if (timer >= updateRate)
+            {
+                 if (!detonated)
+                {
+                    if (timer >= updateRate)

```

Comment:
Generally better to avoid `>=` or `<=` checks.  C# is *likely* to do okay with them, especially with constants involved, but in this case `==` is sufficient.  That said, this would be a good case for `!(timer % updateRate)`, and ten skip the `timer = 0` call.  Whole thing becomes `if (!detonated && !(timer++ % updateRate))`.